pkgs/applications/audio/pd-plugins/gem/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, autoreconfHook,
+  autoconf, automake, libtool, pkgconfig,
+  puredata,
+  libGL, libGLU, glew, freeglut, libv4l, libX11,
+  file
+ }:
+
+stdenv.mkDerivation rec {
+  pname = "gem";
+  version = "unstable-2020-03-26";
```

Comment:
Just to be sure: Your patch doesn't apply on their latest release?

Reply:
The patch applies to the latest release (v0.94), but the build fails. I didn't track down what changed between that version and the current master that fixes the build on Nix.

Reply:
I'd put this info in a comment